web/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Our Node base image
-FROM node:18.16.0-alpine3.18@sha256:9036ddb8252ba7089c2c83eb2b0dcaf74ff1069e8ddf86fe2bd6dc5fecc9492d as base
+FROM node:20.8-alpine3.18@sha256:9036ddb8252ba7089c2c83eb2b0dcaf74ff1069e8ddf86fe2bd6dc5fecc9492d as base
```

Comment:
Is the hash correct? It looks similar to the hash of the previous version

Reply:
I think this one needs to be removed like the other two

Reply:
oops, forgot to remove the hash in the web. fixed now!